staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -411,6 +411,9 @@ func populateVsphereInstanceMap(cfg *VSphereConfig) (map[string]*VSphereInstance
 			klog.Error(msg)
 			return nil, errors.New(msg)
 		}
+		if len(cfg.VirtualCenter) > 1 {
+			klog.Warning("Multi vCenter support is deprecated. vSphere CSI Driver does not support Kubernetes nodes spread across multiple vCenter servers. Please consider moving all Kubernetes nodes to single vCenter server")
+		}
```

Comment:
So we no longer support multiple vcenters but we still support multiple datacenters spread multiple "zones" right?

Reply:
Yes, we do support multiple datacenters. We have validated migration to CSI with zone/region based deployment.